docs: update view session recordings in ui steps (#3421)
hashicorp/boundary: website/content/docs/operations/manage-recorded-sessions.mdx

   ```bash
   boundary session-recordings download -id chr_1234567890
   ```

</Tab>
<Tab heading="Admin Console">

## Find all recorded sessions

You can find all recorded sessions in the UI from the global scope.

1. Log in to Boundary.
1. Select **Session Recordings** in the navigation pane.

   The **Session Recordings** page displays the created time, user, project,
   target, and duration of the recording.

> ## View Session Recording details

Reviewer: ```suggestion
## View session recording details
```

1. Select **View** next to the session recording you want to view.

   The details page has information related to the session recording and
   links to related inforation like user, target, and storage bucket.

> ## Playback Channel Recording

Reviewer: ```suggestion
## Play back channel recording
```

1. Select **Play** next to the channel recording you want to view.

   The recorded session appears in the media player.

>    You can click the **Play** button located at the bottom of the media player

Reviewer: ```suggestion
   Click the **Play** button located at the bottom of the media player
```

   to watch the recording.

>    If a recorded session channel does not support playback a **View** button

Reviewer: ```suggestion
   If a recorded session channel does not support playback, a **View** button
```

>    will be shown. The playback page will display a message stating playback
>    is not supported but still show details specific to that channel.

Reviewer: ```suggestion
   is shown. The playback page displays a message stating that playback
   is not supported, but still shows details specific to that channel.
```
